src/js/components/IconSelect.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <v-select
+        v-model="mutableValue"
+        :options="options"
+        label="title">
```

Comment:
Move the `>` to the next line:

```suggestion
        label="title"
    >
```